src/flows/access/getAccessFlowData.ts

```diff
@@ -27,16 +27,9 @@ const getAccessFlowProps = (): FlowProps => {
       attributesToGet: [
         ...defaultAttributesToGet,
         "children:access-check:jobs",
-        "updateMemberships",
-      ],
-    },
-    {
-      queueName: "update-membership",
-      attributesToGet: [
-        ...defaultAttributesToGet,
         "roleAccesses",
+        "updateMemberships",
         "manageRewards",
-        "shareSocials",
```

Comment:
We don't need shareSocials anymore? Then, why is it present in the AccessLogicParams type?

Reply:
Thanks! I removed it by mistake.